nexus/src/external_api/http_entrypoints.rs

```diff
@@ -6428,34 +6430,107 @@ impl NexusExternalApi for NexusExternalApiImpl {
 
     async fn login_saml_begin(
         rqctx: RequestContext<Self::Context>,
-        path_params: Path<params::LoginToProviderPathParam>,
-        query_params: Query<params::LoginUrlQuery>,
+        _path_params: Path<params::LoginToProviderPathParam>,
+        _query_params: Query<params::LoginUrlQuery>,
     ) -> Result<Response<Body>, HttpError> {
-        console_api::login_saml_begin(rqctx, path_params, query_params).await
```

Comment:
All this function did was call `service_console_index`. That's the kind of thing we're fixing here.